src/AspNet.Security.OAuth.Introspection/Events/AuthenticationFailedContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
```

Comment:
Ah. I think I prefer Allman but it looks like K&R on the wiki is almost the same aside from the variants. *shrug* whatever, I'll move around the silly little `using` statements when I add comments and "prettify" the code :laughing: